fontforgeexe/startui.c

```diff
@@ -203,19 +203,28 @@ static void InsCharHook(GDisplay *gd,unichar_t ch) {
     GInsCharSetChar(ch);
 }
 
-extern GImage splashimage;
+extern GImage splashimage_legacy;
+static GImage *splashimagep;
 static GWindow splashw;
 static GTimer *autosave_timer, *splasht;
 static GFont *splash_font, *splash_italic, *splash_mono;
 static int as,fh, linecnt;
 static unichar_t msg[546];
 static unichar_t *lines[32], *is, *ie;
 
+static void SplashImageInit() {
+    if (splashimagep == NULL)
+        splashimagep = GGadgetImageCache("splash2020.png");
```

Comment:
My one comment is that if you were to name this "spash_special.png" (or something) instead, future releases could have a dedicated splash-screen with no code changes. Not sure if that's worth it or not.

Reply:
I thought about doing it that way. The major deciding factor for me was that it is easier to track text changes than to track binary file replacements. As it is, the regular splash screen is named splash2019.png, for example, so switching back is as easy as a one-line patch of two characters.